packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -402,47 +402,40 @@ function Wayfinding({ onStartDirections, onBack, directionsToLocation, direction
     return (
         <div className="wayfinding" ref={wayfindingRef}>
             <div className="wayfinding__directions">
-                <div className="wayfinding__title">{t('Start wayfinding')}</div>
+                <div className="wayfinding__title">{t('Directions')}</div>
                 <button className="wayfinding__close"
                     onClick={() => closeWayfinding()}
                     aria-label="Close">
                     <CloseIcon />
                 </button>
                 <div className="wayfinding__locations">
-                    <label className="wayfinding__label">
-                        {t('From').toUpperCase()}
-                        <SearchField
-                            ref={fromFieldRef}
-                            mapsindoors={true}
-                            google={selectedMapType === mapTypes.Google && searchExternalLocations}
-                            mapbox={selectedMapType === mapTypes.Mapbox && searchExternalLocations}
-                            placeholder={t('Search by name, category, building...')}
-                            results={locations => searchResultsReceived(locations, searchFieldIdentifiers.FROM)}
-                            clicked={() => onSearchClicked(searchFieldIdentifiers.FROM)}
-                            cleared={() => onSearchCleared(searchFieldIdentifiers.FROM)}
-                            changed={() => onInputChanged(searchFieldIdentifiers.FROM)}
-                        />
-                    </label>
+                    <SearchField
+                        ref={fromFieldRef}
```

Comment:
Now that the explicit label was removed, consider adding an `aria-label` attribute on the inputs.